test/e2e/vrfstest/vrf_create_test.go

```diff
@@ -0,0 +1,87 @@
+package vrfstest
+
+import (
+	"io"
+	"os"
+	"strings"
+	"testing"
+
+	root "github.com/equinix/metal-cli/internal/cli"
+	outputPkg "github.com/equinix/metal-cli/internal/outputs"
+	"github.com/equinix/metal-cli/internal/vrf"
+	"github.com/equinix/metal-cli/test/helper"
+	"github.com/spf13/cobra"
+)
+
+func TestCli_Vrf_Create(t *testing.T) {
+	var (
+		projectId string
+		err       error
+		connId    string
+		vlanId    string
+		portID    string
+	)
+
+	subCommand := "vrf"
+	consumerToken := ""
+	apiURL := ""
+	Version := "metal"
+	rootClient := root.NewClient(consumerToken, apiURL, Version)
+	type fields struct {
+		MainCmd  *cobra.Command
+		Outputer outputPkg.Outputer
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		want    *cobra.Command
+		cmdFunc func(*testing.T, *cobra.Command)
+	}{
+		{
+			name: "create-vrf",
+			fields: fields{
+				MainCmd:  vrf.NewClient(rootClient, outputPkg.Outputer(&outputPkg.Standard{})).NewCommand(),
+				Outputer: outputPkg.Outputer(&outputPkg.Standard{}),
+			},
+			want: &cobra.Command{},
+			cmdFunc: func(t *testing.T, c *cobra.Command) {
+				root := c.Root()
+				projectId, err = helper.CreateTestProject("metal-cli-create-vrf")
+				if err != nil {
+					t.Error(err)
+				}
+
+				if len(projectId) != 0 {
+
+					if len(connId) != 0 && len(vlanId) != 0 && len(portID) != 0 {
```

Comment:
Do we need this check?  The test should fail if these aren't set, so we should either (1) not check the values before executing the command or (2) have an `else` that throws an error so that we know the test didn't run successfully.